package.json

```diff
@@ -19,35 +19,24 @@
   },
   "devDependencies": {
     "husky": "^1.3.1",
-    "kcd-scripts": "^11.2.0",
-    "lint-staged": "^8.1.0",
+    "kcd-scripts": "^12.3.0",
+    "lint-staged": "^13.0.3",
     "prettier": "^2.2.1"
   },
   "lint-staged": {
-    "linters": {
-      "*.js": [
-        "prettier --write",
-        "git add"
-      ],
-      "*.mdx": [
-        "prettier --write",
-        "git add"
-      ],
-      "*.json": [
-        "prettier --write ",
-        "git add"
-      ]
-    }
+    "*.js": "prettier --write",
```

Comment:
Doing `git add` isn't required anymore - https://github.com/okonet/lint-staged#v10 

> From v10.0.0 onwards any new modifications to originally staged files will be automatically added to the commit. If your task previously contained a git add step, please remove this. The automatic behaviour ensures there are less race-conditions, since trying to run multiple git operations at the same time usually results in an error.

Reply:
Cool 👍